build.sbt

```diff
@@ -264,6 +264,7 @@ lazy val V = new {
   val bloop = "1.4.11-17-f4ef8273"
```

Comment:
Let's bump bloop to the newest snapshot to see if it's working. I should release 1.4.12 version for sure also for the Test Explorer.